content/en/docs/OIDC/identity_providers/gitlab.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to configure Identity provider for Gitlab
```

Comment:
I'd say "GitLab as an Identity Provider".

We configure GitLab as an IdP for Keycloak here, though it's also possible to configure Keycloak as an IdP for GitLab, which is the opposite.

The same for other providers.